src/Xamarin.Android.Tools.AndroidSdk/JdkInfo.cs

```diff
@@ -144,21 +144,25 @@ Version GetJavaVersion ()
 
 		ReadOnlyDictionary<string, string> GetReleaseProperties ()
 		{
+			var props       = new Dictionary<string, string> ();
 			var releasePath = Path.Combine (HomePath, "release");
 			if (!File.Exists (releasePath))
-				return new ReadOnlyDictionary<string, string> (new Dictionary<string, string> ());
+				return new ReadOnlyDictionary<string, string>(props);
 
-			var props   = new Dictionary<string, string> ();
 			using (var release = File.OpenText (releasePath)) {
 				string line;
 				while ((line = release.ReadLine ()) != null) {
-					const string PropertyDelim  = "=\"";
+					const string PropertyDelim  = "=";
 					int delim = line.IndexOf (PropertyDelim, StringComparison.Ordinal);
 					if (delim < 0) {
 						props [line] = "";
+						continue;
 					}
 					string  key     = line.Substring (0, delim);
-					string  value   = line.Substring (delim + PropertyDelim.Length, line.Length - delim - PropertyDelim.Length - 1);
+					string  value   = line.Substring (delim + PropertyDelim.Length);
```

Comment:
Should consider any other weirdness that could happen when parsing this text?

Maybe a call to `Trim ()` for the original `key` and `value`? Such as:
```
KEY_A = 1234 
KEY_B = "Some text here"
```